src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,34 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
```

Comment:
quick note, if you know range starts at zero, you dont need to include it, also stopping index is non-inclusive